### ISSUE TITLE
Error building vignettes

### ISSUE THREAD

**Issue by mg14** [gerstung-lab/deepSNV#17]
Bioconductor currently fails to build the current release due to an issue with reading the VCF headers when building the vignettes.

```
##############################################################################
##############################################################################
###
### Running command:
###
###   /home/biocbuild/bbs-3.16-bioc/R/bin/R CMD build --keep-empty-dirs --no-resave-data deepSNV
###
##############################################################################
##############################################################################


* checking for file ‘deepSNV/DESCRIPTION’ ... OK
* preparing ‘deepSNV’:
* checking DESCRIPTION meta-information ... OK
* cleaning src
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘shearwaterML.Rmd’ using rmarkdown
[E::bcf_hdr_parse_sample_line] Could not parse the "#CHROM.." line, either FORMAT is missing or spaces are present instead of tabs:
	#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT

Quitting from lines 35-39 (shearwaterML.Rmd) 
Error: processing vignette 'shearwaterML.Rmd' failed with diagnostics:
error in evaluating the argument 'x' in selecting a method for function 'as.list': error in evaluating the argument 'x' in selecting a method for function 'as.list': no 'header' line "#CHROM POS ID..."?
--- failed re-building ‘shearwaterML.Rmd’

--- re-building ‘deepSNV.Rnw’ using knitr
--- finished re-building ‘deepSNV.Rnw’

--- re-building ‘shearwater.Rnw’ using knitr
[E::bcf_hdr_parse_sample_line] Could not parse the "#CHROM.." line, either FORMAT is missing or spaces are present instead of tabs:
	#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT

Quitting from lines 348-353 (shearwater.Rnw) 
Error: processing vignette 'shearwater.Rnw' failed with diagnostics:
error in evaluating the argument 'x' in selecting a method for function 'as.list': error in evaluating the argument 'x' in selecting a method for function 'as.list': no 'header' line "#CHROM POS ID..."?
--- failed re-building ‘shearwater.Rnw’

SUMMARY: processing the following files failed:
  ‘shearwaterML.Rmd’ ‘shearwater.Rnw’

Error: Vignette re-building failed.
Execution halted
```

**Comment by jmarshall** [comment]
Ah — that's a relatively recent error added in htslib 1.14's VCF parsing: `FORMAT` should only be there if there actually are some sample columns after it. (The message is misleading in this context — what it's looking for is `\tFORMAT\t`.)

Hopefully we can just remove `\tFORMAT` from those lines in the template VCF files, and when VariantAnnotation APIs are used to add named samples later, they should automatically add `FORMAT` back too.

**Comment by jmarshall** [comment]
Fixed by 3b904c4adb49ed6d0b3e61cb19fd6101b7cadc3d.
